membersrvc/ca/ca.go

```diff
@@ -33,7 +33,7 @@ import (
 
 	"github.com/hyperledger/fabric/core/crypto/primitives"
 	pb "github.com/hyperledger/fabric/membersrvc/protos"
-	_ "github.com/mattn/go-sqlite3"
+	_ "github.com/mattn/go-sqlite3" // TODO: not use a blank import!
```

Comment:
Looks good. Agree it should be justified to help people understand why it was used.